### ISSUE TITLE
Project access rights based on GitHub groups

### ISSUE THREAD

**Issue by jirikuncar** [quantifiedcode/issues#13]
Is it possible to allow people with admin rights to a project on GitHub modify settings on QC?

**Comment by programmdesign** [comment]
HI @jirikuncar. Sorry for the late response. The release of QC 2.0 kept us too busy. 

To your question: yes, this is possible. Just go to your project settings, click on "Access Management" and invite Collaborators as Admins. 

Does that answer your question?

**Comment by jirikuncar** [comment]
Maybe I should have asked differently. Is it possible to keep the rights automatically in sync with the groups on GitHub so we don't have to manage teams on two sites (similarly how Travis CI is doing it).

**Comment by programmdesign** [comment]
No, not yet unfortunately. But we do have that on our feature list.

**Comment by jirikuncar** [comment]
Thanks, I'm looking forward to see this feature in QC 3.0 :wink:
